### PR TITLE
Remove path conf

### PR DESCRIPTION
**Pull request by yyongycy**
### What changes are proposed in this pull request?

Remove path conf.

### Why are the changes needed?

Remove path conf.

### Does this PR introduce any user facing changes?

NA

### CODE REVIEW

**Comment by yyongycy** [comment]
@dbw9580 please help take a look

File: dora/core/client/fs/src/main/java/alluxio/client/file/DoraCacheFileSystem.java [hunk 1]
```diff
@@ -137,7 +137,7 @@ public URIStatus getStatus(AlluxioURI path, GetStatusPOptions options)
     }
     try {
       GetStatusPOptions mergedOptions = FileSystemOptionsUtils.getStatusDefaults(
-          mFsContext.getPathConf(path)).toBuilder().mergeFrom(options).build();
+          mFsContext.getClusterConf()).toBuilder().mergeFrom(options).build();
```
Comment (apc999) [inline dora/core/client/fs/src/main/java/alluxio/client/file/DoraCacheFileSystem.java:140]: I am not quite sure if simply replacing `getPathConf` by `getClusterConf` (a pattern in this PR) is always correct.
Can you double-check it?

**Review by apc999 — CHANGES_REQUESTED** [review]

**Review by apc999 — APPROVED** [review]
LGTM modular my comment. Just double check it if we are introducing any behavior different

File: dora/core/client/fs/src/main/java/alluxio/client/file/DoraCacheFileSystem.java (hunk 1 above)
Comment (yyongycy) [reply dora/core/client/fs/src/main/java/alluxio/client/file/DoraCacheFileSystem.java:140]: With the recent change, there is no configure propagation, everything depends on the local config, there should be no difference.

**Comment by yyongycy** [comment]
alluxio-bot, merge this please

**Pull request merged**